src/components/UserSelector.tsx

```diff
@@ -1,16 +1,77 @@
-import React from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { User } from '../types/User';
+import classnames from 'classnames';
+import { getUsers } from '../api/users';
+
+type Props = {
+  selectedUser: User | null;
```

Comment:
According to the checklist, make sure you described objects in propTypes. Here, the `User` object is not described in propTypes.